lib/prompts/__tests__/downloadProjectPrompt.test.ts

```diff
@@ -1,21 +1,19 @@
 // @ts-nocheck
+const { getAccountId } = require('@hubspot/local-dev-lib/config');
```

Comment:
I did a little bit of moving around of imports and files in the lib folder just to make things consistent. Some of these tests still accounted for the old folder structure of the repo where CLI was one of three packages